src/main/java/mate/academy/bookstorev2/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,64 @@
+package mate.academy.bookstorev2.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Positive;
+import lombok.RequiredArgsConstructor;
+import mate.academy.bookstorev2.dto.cart.item.CartItemAddBookResponseDto;
+import mate.academy.bookstorev2.dto.cart.item.CartItemUpdateDto;
+import mate.academy.bookstorev2.dto.shopping.cart.AddToShoppingCartDto;
+import mate.academy.bookstorev2.dto.shopping.cart.ShoppingCartDto;
+import mate.academy.bookstorev2.model.User;
+import mate.academy.bookstorev2.service.ShoppingCartService;
+import org.springdoc.core.annotations.ParameterObject;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Shopping Cart management", description = "Endpoints for managing shopping carts")
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/cart")
```

Comment:
```suggestion
@RequestMapping("/cart")
```

add `server.servlet.context-path=/api` to application.properties